notebooks/official/model_monitoring/batch_prediction_model_monitoring.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>why not just do</p><p> </p><p>! gsutil cat SKEW_GS_PATH&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>fixed</p>